### PR TITLE
First layout test

### PR DESCRIPTION
**Pull request by Dorian30**
## Summary
- I built the 4 components for the layout site and put them together.

## Screenshot
layout

## Trello Card
https://trello.com/c/Akp0W85v/11-setup-del-proyecto

### CODE REVIEW

File: maquetado/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (CarlosHWolox) [inline maquetado/.vscode/settings.json]: You shouls ignore this file aswell as it is custom for your editor

File: maquetado/index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (CarlosHWolox) [inline maquetado/index.html]: We use two spaces for indentation

File: maquetado/index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    <div class="footer-logo">
+      <span class="title-big footer-slogan">we are game changers</span>
+      <img src="img/wolox_logo_white.svg" alt="wolox white logo">
+    </div>
+    <p class="title-small footer-copyright">
+      Copyright 2018. Wolox. All rights reserverd.
+    </p>
+  </footer>
+</body>
+</html>
```
Comment (CarlosHWolox) [inline maquetado/index.html]: We leave an empty line at the end of file so this icon does not show

File: maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.nav {
+  align-items: center;
+  background: white;
+  box-shadow: 0px 5px 10px $grey;
+  display: flex;
+  height: 90px;
```
Comment (CarlosHWolox) [inline maquetado/scss/navbar.scss]: This height is defined as a constant on the training. Use it as a variable

File: maquetado/scss/styles.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+.main-content {
+  padding: 160px 100px 100px;
+
+  .tech-area:not(:last-child){
+    margin: 0 0 90px;
+  }
+}
+
+.tech-area{
+
```
Comment (CarlosHWolox) [inline maquetado/scss/styles.scss]: No empty line

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    transition: all 0.5s ease;
+    width: 100%;
+    z-index: -1;
+  }
+
+  &:hover {
+
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: no empty lines.

**Review by CarlosHWolox — APPROVED** [review]

File: maquetado/.gitignore [hunk 7]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (CarlosHWolox) [inline maquetado/.gitignore]: Leave an empty line at end of file

File: maquetado/application.css [hunk 8]
```diff
@@ -0,0 +1,236 @@
+* {
```
Comment (CarlosHWolox) [inline maquetado/application.css]: This file should be added to the .gitignore

File: maquetado/scss/colors.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
+$blue: #00ADEE;
+$green: #BED23C;
+$grey: #C4C4C4;
```
Comment (CarlosHWolox) [inline maquetado/scss/colors.scss]: leave a empty line at end of all files

**Review by CarlosHWolox — COMMENTED** [review]
There should not appear the application.css.map

**Review by CarlosHWolox — APPROVED** [review]

File: maquetado/index.html [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+</head>
+<body>
+  <nav class="nav">
+    <img src="img/wolox_logo.svg" alt="Wolox logo" class="nav-logo">
+    <h1 class="title-big">Developers Tools</h1>
+  </nav>
+  <main class="main-content">
+    <div class="tech-area">
+      <div class="title-container">
+        <img class="title-decoration" src="img/triangle.svg" alt="title decoration">
```
Comment (mcallegari10) [inline maquetado/index.html]: For images that are just for a "decoration" purpose, they are not really important semantically for our HTML structure, we use [pseudo-elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements). Please check that and let me know if you have any questions 😄

File: maquetado/index.html [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML & CSS Training</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="application.css">
+</head>
+<body>
+  <nav class="nav">
+    <img src="img/wolox_logo.svg" alt="Wolox logo" class="nav-logo">
```
Comment (mcallegari10) [inline maquetado/index.html]: No need for the word `logo` in the `alt`. Keep in mind this attribute should only be a description of the image, not the actual image usage

File: maquetado/index.html [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+          <p class="tech-box-release">Release: 2005</p>
+          <p class="tech-box-info">Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.</p>
+        </div>        
+      </div>
+    </div>
+  </main>
+  <footer class="footer">
+    <div class="footer-logo">
+      <span class="title-big footer-slogan">we are game changers</span>
+      <img src="img/wolox_logo_white.svg" alt="wolox white logo">
```
Comment (mcallegari10) [inline maquetado/index.html]: Same here, no need for the logo

File: maquetado/index.html [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+  </nav>
+  <main class="main-content">
+    <div class="tech-area">
+      <div class="title-container">
+        <img class="title-decoration" src="img/triangle.svg" alt="title decoration">
+        <h3 class="title-big">Front-End Tools</h3>
+      </div> 
+      <div class="tech-list">
+        <div class="tech-box">
+          <img src="img/react_icon.svg" alt="React logo" class="tech-box-logo">
```
Comment (mcallegari10) [inline maquetado/index.html]: Same here, check the alt attribute and remove the logo. Check the other techs! 😄

File: maquetado/scss/styles.scss [hunk 14]
```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-content {
+  padding: 160px 100px 100px;
```
Comment (mcallegari10) [inline maquetado/scss/styles.scss]: No need for this. If you want to set a space between the content and the border of the browser, you can do something like:

```scss
.main-content {
  margin: 0 auto;
  max-width: $content-size: // this content size should be something like 800px
  width: 100%:
}
```
That way, you content will be always on the center of the screen as we want it 😄

File: maquetado/scss/styles.scss [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+
+  .tech-area:not(:last-child){
```
Comment (mcallegari10) [inline maquetado/scss/styles.scss]: No need for this

File: maquetado/scss/styles.scss [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+    margin: 0 0 90px;
+  }
+}
+
+.tech-area{
+  .tech-list {
+    margin: 20px 0 0 0;
```
Comment (mcallegari10) [inline maquetado/scss/styles.scss]: I think this is not needed and it's solved with the code in the `main-content`

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: maquetado/index.html (hunk 10 above)
Comment (Dorian30) [reply maquetado/index.html]: Ok, I understand. I'll change it to a pseudo-element.

File: maquetado/index.html (hunk 11 above)
Comment (Dorian30) [reply maquetado/index.html]: Ok. I'm on it.

File: maquetado/scss/styles.scss (hunk 14 above)
Comment (Dorian30) [reply maquetado/scss/styles.scss]: Perfect! I was not sure if the content was supposed to be centered in this case. I'll change it.

File: maquetado/scss/styles.scss (hunk 16 above)
Comment (Dorian30) [reply maquetado/scss/styles.scss]: @mcallegari10 I'm using this to make some space between the title area and the box list where all cards sit. 

image

Do I delete it anyway ?

File: maquetado/scss/styles.scss (hunk 15 above)
Comment (Dorian30) [reply maquetado/scss/styles.scss]: Ok. I'll delete it.

File: maquetado/scss/footer.scss [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+
+.footer-logo {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.footer-slogan {
+  color: white;
```
Comment (mcallegari10) [inline maquetado/scss/footer.scss]: Change to variable please

File: maquetado/scss/footer.scss [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+  margin: 0 0 15px;
+  text-transform: uppercase;
+}
+
+.footer-copyright {
+  color: white;
```
Comment (mcallegari10) [inline maquetado/scss/footer.scss]: Same here

File: maquetado/scss/footer.scss [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+  display: block;
```
Comment (mcallegari10) [inline maquetado/scss/footer.scss]: I think this is not needed, check the alignment with `align-self`

File: maquetado/scss/titles.scss [hunk 20]
```diff
@@ -0,0 +1,29 @@
+.tech-area-title {
+  align-items: center;
+  display: flex;
+  margin: 0 0 30px;
+  position: relative;
```
Comment (mcallegari10) [inline maquetado/scss/titles.scss:5]: If the pseudo-element has position `relative`, there's no need to add this relative 😄

File: maquetado/scss/application.scss [hunk 21]
```diff
@@ -0,0 +1,17 @@
+$content-size: 1200px;
+$nav-height: 90px;
+$nav-logo-height: 30px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+$tech-logo-size: 100px;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
```
Comment (mcallegari10) [inline maquetado/scss/application.scss]: I'd put this variables in a file like `colors` but name it `sizes`

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**